homeassistant/components/risco/binary_sensor.py

```diff
@@ -3,23 +3,76 @@
 from __future__ import annotations
 
 from collections.abc import Mapping
+from itertools import chain
 from typing import Any
 
 from pyrisco.cloud.zone import Zone as CloudZone
+from pyrisco.common import System
 from pyrisco.local.zone import Zone as LocalZone
 
 from homeassistant.components.binary_sensor import (
     BinarySensorDeviceClass,
     BinarySensorEntity,
+    BinarySensorEntityDescription,
 )
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import LocalData, RiscoDataUpdateCoordinator, is_local
-from .const import DATA_COORDINATOR, DOMAIN
+from .const import DATA_COORDINATOR, DOMAIN, SYSTEM_UPDATE_SIGNAL
 from .entity import RiscoCloudZoneEntity, RiscoLocalZoneEntity
 
+SYSTEM_ENTITY_DESCRIPTIONS = [
+    BinarySensorEntityDescription(
+        key="low_battery_trouble",
+        name="Low battery trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="ac_trouble",
+        name="A/C trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="monitoring_station_1_trouble",
+        name="Monitoring station 1 trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="monitoring_station_2_trouble",
+        name="Monitoring station 2 trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="monitoring_station_3_trouble",
+        name="Monitoring station 3 trouble",
```

Comment:
Is trouble a word used within the risco system?

Reply:
Yes, that's how it's described everywhere in the Risco ecosystem